src/MagentoHackathon/Composer/Magento/DeployManager.php

```diff
@@ -97,7 +100,30 @@ public function doDeploy()
                     $this->io->write($e->getMessage());
                 }
             }
+        }
+    }
+
+    /**
+     * Determine the priority in which the package should be deployed
+     *
+     * @param Entry $package
+     * @return int
+     */
+    private function getPackagePriority(Entry $package)
+    {
+        $result = 100;
+        $maxPriority = max(array_merge($this->sortPriority, [100, 101]));
 
+        if (isset($this->sortPriority[$package->getPackageName()])) {
+            $result = $this->sortPriority[$package->getPackageName()];
+        } elseif (isset($this->highPriority[$package->getPackageName()])) {
```

Comment:
The highPriority check should be first so it's used even if the package appears in both highPriority and sortPriority.

Reply:
Makes sense. Change made.